### DISCUSSION TITLE
How can I receive a photo and immediately send it without saving it back to the user?

### DISCUSSION THREAD

**Discussion by greatUzumaki** [aiogram/aiogram#1393] [Q&A]
Hello everyone, I know that this question has already been asked, but it’s not clear how to use BytesIO
Are there any examples?

```python
async def handle_finish(message: Message, state: FSMContext):
    data = await state.get_data()

    document: List[PhotoSize] | Document = data['WaitingForDocument']
    selfie: List[PhotoSize] | Document = data['WaitingForSelfie']
    address: str = data['WaitingForAddress']

    if type(document) is Document:
        downloaded = await bot.download(document)
        await message.answer_document(downloaded)
    else:
        downloaded = await bot.download(document[-1])
        await message.answer_photo(InputFile(downloaded))
  

    await message.answer("Передаю данные менеджеру! Скоро он всё проверит")
    await bot.send_media_group(CHANNEL, media=[])
   ```

I'm just trying to get user submitted files (document and selfie) to send them as a group

**Comment by JrooTJunior** [comment]
You don't need to download file to sent in back to the Telegram, you can use file_id instead or just use copy method.

Using file_id:
```python
await bot.send_photo(chat_id=..., photo=message.photo[-1].file_id)
```
(or `message.answer_photo(...)`)

Using copy method from API https://core.telegram.org/bots/api#copymessage
with using this method you don't need to check that the message is photo or document, just only check that it can be copied
